packages/client/src/client/App/Analytics/AnalyticsCore.tsx

```diff
@@ -1,37 +1,52 @@
-import { Subscribe } from "@react-rxjs/core"
 import { merge } from "rxjs"
+import styled from "styled-components"
 
-import { supportsTearOut } from "@/client/App/TearOutSection/supportsTearOut"
-import { TearOutComponent } from "@/client/App/TearOutSection/TearOutComponent"
+import { Region } from "@/client/components/layout/Region"
+import { TabBar, TabBarActionConfig } from "@/client/components/TabBar"
 import { createSuspenseOnStale } from "@/client/utils/createSuspenseOnStale"
 import { WithChildren } from "@/client/utils/utilityTypes"
 import { isAnalyticsDataStale$ } from "@/services/analytics"
 
+import { supportsTearOut } from "../TearOutSection/supportsTearOut"
+import { TearOutComponent } from "../TearOutSection/TearOutComponent"
 import { PnL, pnL$ } from "./PnL"
 import { Positions, positions$ } from "./Positions"
 import { ProfitAndLoss, profitAndLoss$ } from "./ProfitAndLoss"
-import {
-  AnalyticsHeader,
-  AnalyticsInnerWrapper,
-  AnalyticsStyle,
-  RightNav,
-} from "./styled"
+import { AnalyticsStyle } from "./styled"
 
 const analytics$ = merge(pnL$, profitAndLoss$, positions$)
 
 const SuspenseOnStaleData = createSuspenseOnStale(isAnalyticsDataStale$)
 
+const actions: TabBarActionConfig = []
+
+if (supportsTearOut) {
+  actions.push({
+    name: "tearOut",
+    size: "sm",
+    inner: <TearOutComponent section="analytics" />,
+  })
+}
+
+const AnalyticsRegion = styled(Region)`
+  width: 320px;
+  background-color: ${({ theme }) =>
+    theme.newTheme.color["Colors/Background/bg-primary"]};
```

Comment:
thinking about this .. looks awkward .. no immediate actions, but I'd like to do better around color refs .. in RTEq I am doing all the fiddly mapping in `theme.ts`